src/celestia/audiosession.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <celcompat/filesystem.h>
+
+namespace celestia
+{
+
+namespace constants
+{
+const float minAudioPan = -1.0f;
+const float maxAudioPan = 1.0f;
+const float defaultAudioPan = 0.0f;
+const float minAudioVolume = 0.0f;
+const float maxAudioVolume = 1.0f;
+const float defaultAudioVolume = 1.0f;
+const int minAudioChannel = 0;
+const int defaultAudioChannel = 0;
+}
+
+class AudioSession
+{
+ public:
+    explicit AudioSession(const fs::path &path, float volume, float pan, bool loop, bool nopause);
```

Comment:
`explicit` is not required here.
```
1) Specifies that a constructor or conversion function (since C++11) or [deduction guide](https://en.cppreference.com/w/cpp/language/ctad) (since C++17) is explicit, that is, it cannot be used for [implicit conversions](https://en.cppreference.com/w/cpp/language/implicit_cast) and [copy-initialization](https://en.cppreference.com/w/cpp/language/copy_initialization).
...
A constructor with a single non-default parameter (until C++11) that is declared without the function specifier explicit is called a [converting constructor](https://en.cppreference.com/w/cpp/language/converting_constructor).
```

Reply:
yea h I figure so, I added it when the initializer has only one argument, and forgot to delete it.